docs/Installing.asciidoc

```diff
@@ -308,12 +308,14 @@ For a local instance setup you can simply execute the script:
 This will automatically setup a local apache http proxy. Read on for more
 detailed setup instructions with all the details.
 
+If you wish to run openQA behind an http proxy (apache, NGINX, etc..) then see the
+*openqa.conf.template* config file in */etc/apache2/vhosts.d* (openSUSE) or
+`/etc/httpd/conf.d` (Fedora) when using apache2 or the config files in
+`/etc/nginx/vhosts.d` for nginx.
```

Comment:
```suggestion
`/etc/nginx/vhosts.d` for NGINX.
```
or
```suggestion
`/etc/nginx/vhosts.d` for Nginx.
```
It doesn't really matter which one; just be consistent with the spelling one sentence before.